libyul/backends/evm/AbstractAssembly.h

```diff
@@ -101,9 +101,9 @@ class AbstractAssembly
 	/// Creates a new sub-assembly, which can be referenced using dataSize and dataOffset.
 	virtual std::pair<std::shared_ptr<AbstractAssembly>, SubID> createSubAssembly() = 0;
 	/// Appends the offset of the given sub-assembly or data.
-	virtual void appendDataOffset(SubID _sub) = 0;
+	virtual void appendDataOffset(std::vector<SubID> const& _subIdOrPath) = 0;
```

Comment:
```suggestion
	virtual void appendDataOffset(std::vector<SubID> const& _subIdPath) = 0;
```
(a single id is also a path)